components/Form.js

```diff
@@ -1,26 +1,66 @@
 import styled from "styled-components";
 import PlacesAutocomplete from "./PlacesAutoComplete";
+import {Slider, SliderMarker} from "@reach/slider";
+import "@reach/slider/styles.css";
 
-export function FormHome({setSelected, setSelectedDest}) {
+export function FormHome({
+  setSelected,
+  setSelectedDest,
+  radius,
+  setRadius,
+  stops,
+  setStops,
+}) {
+  const handleChangeStops = newValue => {
+    setStops(newValue);
+  };
+
+  const handleChangeRadius = newValue => {
+    setRadius(newValue);
+  };
   return (
-    <FormContainer method="get">
-      <span>Starting Point:</span>
-      <PlacesAutocomplete setSelected={setSelected} positionStatus={"Start"} />
-      <span>Destination:</span>
-      <PlacesAutocomplete
-        setSelectedDest={setSelectedDest}
-        positionStatus={"Dest"}
-      />
-      <span>Amount of stops:</span>
-      <label htmlFor="stops">
-        <input type="number" name="stops" id="stops" required />
-      </label>
-      <span>Set radius:</span>
-      <label htmlFor="radius">
-        <input type="number" name="radius" id="radius" required />
-      </label>
-      <StyledButton type="submit">Submit</StyledButton>
-    </FormContainer>
+    <>
```

Comment:
I think you don't need a Fragment at this point.

Reply:
True! I removed it.